README.md

```diff
@@ -29,7 +29,7 @@ Below are Microsoft's official MCP server implementations:
 
 ### 🔷 Azure MCP Server
 
-- **Repository**: [microsoft/mcp](https://github.com/microsoft/mcp/tree/main/servers/Azure.Mcp.Server#readme)
```

Comment:
Microsoft/mcp is intentional since the repo was moved last Thursday. 

Multiple servers will be cohabitating out of the same microsoft/mcp repo (both Microsoft and Azure branded; anything owned by Microsoft), starting with the Azure MCP and Microsoft Fabric MCP servers. Microsoft SQL MCP server is probably onboarding next after Fabric is settled.

Reply:
Missed it! thank you for the update. closing.